app/feature/feature-login/src/main/kotlin/com/hedvig/android/feature/login/marketing/MarketingDestination.kt

```diff
@@ -183,7 +185,7 @@ private fun MarketingScreen(
             modifier = Modifier.testTag("login_button"),
           )
           HedvigTextButton(
-            text = "Get a price quote",
```

Comment:
Wops. Good find.

Reply:
Yeah, I have no idea how this one slipped through, I see that I did some refactoring some months ago and I seem to have just never updated the localization, oops.